mcfix_product/data/product_demo.xml

```diff
@@ -1,37 +1,154 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <data noupdate="1">
+    <data>
```

Comment:
I think you should not remove the noupdate 🤔

Reply:
I think that demo is only updated on installation, on upgrades is not executed.

Reply:
If you update a module with --test-enable, the demo files are also executed. Thus the <code>noupdate=1</code> guarantee that this data is not upgraded.